release-notes/opensearch-dashboards-queryInsightsDashboards.release-notes-3.2.0.0.md

```diff
@@ -0,0 +1,22 @@
+## Version 3.2.0 Release Notes
+
+Compatible with OpenSearch and OpenSearch Dashboards version 3.2.0
+
+### Enhancements
+* [IMPROVEMENT] MDS support for Inflight Queries ([#217](https://github.com/opensearch-project/query-insights-dashboards/pull/217))
+
+### Bug Fixes
+* Fix-query-details-verbose-param ([#217](https://github.com/opensearch-project/query-insights-dashboards/pull/217))
+* [Fix] Ensure accurate time filtering for Top Queries in OpenSearch 2.19+ ([#219](https://github.com/opensearch-project/query-insights-dashboards/pull/219))
+* Revert "[Fix] Ensure accurate time filtering for Top Queries in OpenSearch 2.19+" ([#273](https://github.com/opensearch-project/query-insights-dashboards/pull/273))
```

Comment:
Maybe remove this @dzane17 ?
Adding and reverting the PR does not make sense in release notes IMO.

Reply:
Sure I'll remove it